node/malus/src/variants/suggest_garbage_candidate.rs

```diff
@@ -80,171 +71,188 @@ where
 {
 	type Message = CandidateBackingMessage;
 
-	/// Intercept incoming `Second` requests from the `collator-protocol` subsystem. We take
+	/// Intercept incoming `Second` requests from the `collator-protocol` subsystem.
 	fn intercept_incoming(
 		&self,
 		subsystem_sender: &mut Sender,
 		msg: FromOrchestra<Self::Message>,
 	) -> Option<FromOrchestra<Self::Message>> {
 		match msg {
 			FromOrchestra::Communication {
-				msg: CandidateBackingMessage::Second(relay_parent, candidate, _pov),
+				msg: CandidateBackingMessage::Second(relay_parent, ref candidate, ref _pov),
 			} => {
-				gum::debug!(
+				gum::info!(
 					target: MALUS,
-					candidate_hash = ?candidate.hash(),
-					?relay_parent,
-					"Received request to second candidate"
+					"😈 Started Malus node with a {:?} % chance to behave maliciously for a given candidate.",
```

Comment:
This is still printed on each `Second` message we intercept. We should print this information at startup only.